docs/components/top_app_bars.md

```diff
@@ -0,0 +1,351 @@
+<!--docs:
+title: "Material Top App Bar"
+layout: detail
+section: components
+excerpt: "A customizable top app bar component with updated visual styles."
+iconId:
+path: /catalog/material-top-app-bar/
+-->
+
+**Instructions to authors:**
+* _Follow the instructions in each section &ndash; see the [accompanying examples](button-examples) for futher guidance._
+* _Delete these instructions before submitting your document_
+
+# Top app bar
+
+The [top app bar](https://material.io/components/app-bars-top/#) displays information and actions relating to the current screen.
+
+There are two types of top app bar:
+
+1. [Regular top app bar](#regular-top-app-bar)
+1. [Contextual action bar](#contextual-top-app-bar)
+<br>
+
+![Regular app bar: purple background, white text and icons](assets/top_app_bars/Top-app-bars_types_side-by-side.png)
```

Comment:
The alt-text should mention that the image shows both regular and contextual.

Reply:
Done